omf/fileio/geoh5.py

```diff
@@ -41,38 +63,57 @@ def entity(self):
 
     @entity.setter
     def entity(self, element):
-        if type(element) not in _CLASS_MAP:
-            raise ValueError("Element of type {type(element)} currently not implemented.")
+        if type(element) not in _CONVERSION_MAP:
+            raise ValueError(
```

Comment:
should it raise a OMFtoGeoh5Implementation exception ?

Reply:
OMFtoGeoh5NotImplemented